### PR TITLE
converted to module

### PR DESCRIPTION
**Pull request by ctrlbold**
renamed files and functions to conform with PowerShell rules, added auto Internet installer, added module manifest

### CODE REVIEW

**Comment by ctrlbold** [comment]
I wanted to merge with the other repo but there were some conflicts and I'm no good at managing conflicts in Github.

There are a few things that you will have to address, including the name of the module. The directory name should be the module name. So in this case, your module name is Reporting-Services-PowerShell. As you may be familiar, this is an unusual name for a module. If you decide to change the name of the module you will need to update

- [ ] the repo name
- [ ] the psd1 name
- [ ] the psm1 name
- [ ] the psm1 path inside of the psd1
- [ ] the paths (there are like 5) inside of installer.ps1

If the directory name and the psd1 (and perhaps the psm1) names do not match, PowerShell will not automatically import the module even if it's in the user's path (it didn't in my testing at least)

Next, please review the contents of the psd1. This includes the module version (right now at 0.0.0.1),  the icon, your licenses and so on.

Once you approve the PR and make all of your changes, your module will be ready to publish to the PowerShell Gallery 👍

File: Docs/Get-Started-With-RVC.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Next steps - READ!
```
Comment (jtarquino) [inline Docs/Get-Started-With-RVC.md]: would you mind to remove the docs folder (not relevant for this repo)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```
Comment (jtarquino) [inline LICENSE]: Can you remove it , there is a license.md , lets keep only one

File: Install.ps1 [hunk 3]
```diff
@@ -0,0 +1,102 @@
+[CmdletBinding()]
+param (
```
Comment (jtarquino) [inline Install.ps1]: this installing is super cool!!!

File: Reporting-Services-PowerShell.psd1 [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        Category = "Databases"
+
+        # Keyword tags to help users find this module via navigations and search.
+        Tags = @('sqlserver','reportingservices','sql','dba','databases','reporting')
+
+        # The web address of an icon which can be used in galleries to represent this module
+        IconUri = ""
+
+        # The web address of this module's project or support homepage.
+        ProjectUri = ""
```
Comment (jtarquino) [inline Reporting-Services-PowerShell.psd1]: i guess this should be https://github.com/Microsoft/Reporting-Services-PowerShell/

File: Reporting-Services-PowerShell.psd1 [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+        # The web address of this module's license. Points to a page that's embeddable and linkable.
+        LicenseUri = ""
```
Comment (jtarquino) [inline Reporting-Services-PowerShell.psd1]: and this https://raw.githubusercontent.com/Microsoft/Reporting-Services-PowerShell/master/License.md

**Comment by jtarquino** [comment]
I know about merge is the burden of the daily developer ☺, I know the module name will be kind of weird but for now is my best try for a repo name that makes sense , as usual if we found it too annoying we can change latter.

File: Install.ps1 (hunk 3 above)
Comment (ctrlbold) [inline Install.ps1]: :D I had to guess at the URL's and stuff so I may need to redo it but once it works, the module can be installed by executing:

`Invoke-Expression (Invoke-WebRequest https://git.io/vP4dm)`

I put that link in my readme.md for [dbatools](https://github.com/sqlcollaborative/dbatools)

**Pull request merged**